account-kit/react/src/hooks/useAuthenticate.ts

```diff
@@ -30,7 +30,7 @@ export type UseAuthenticateResult = {
  * ```ts
  * import { useAuthenticate } from "@account-kit/react";
  *
- * const { authenticate, authenticateAsync, isPending, error } = useAuthenticate({
+ * const { authenticate, authenticateAsync, isPending, error, stage } = useAuthenticate({
```

Comment:
stage shouldn't be here anymore right?

Reply:
This was removed, I think it was accidentally introduced from a rebase.